pluginconfig/commit.go

```diff
@@ -62,6 +62,10 @@ func (c CommitPluginConfig) Validate() error {
 	return c.OffchainConfig.Validate()
 }
 
+type FeeInfo struct {
+	ExecDeviationPPB   cciptypes.BigInt `json:"execDeviationPPB"`
+	DataAvDeviationPPB cciptypes.BigInt `json:"dataAvailabilityDeviationPPB"`
```

Comment:
nit: Struct field name matches json tag name ideally

```suggestion
	DataAvailabilityDeviationPPB cciptypes.BigInt `json:"dataAvailabilityDeviationPPB"`
```